source/_components/here_travel_time.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: page
+title: "HERE Travel Time"
+description: "Instructions on how to add HERE travel time to Home Assistant."
+date: 2019-06-06 15:06
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: HERE_logo.svg
+ha_category:
+  - Transport
+ha_iot_class: Cloud Polling
+ha_release: 0.95
+redirect_from:
+ - /components/sensor.here_travel_time/
+---
+
+The `here_travel_time` sensor provides travel time from the [HERE Routing API](https://developer.here.com/documentation/routing/topics/introduction.html).
+
+## {% linkable_title Setup %}
+
+You need to register for an API key by following the instructions [here](https://developer.here.com/documentation/routing/topics/introduction.html?create=Freemium-Basic&keepState=true&step=account).
+
+HERE offers a Freemium Plan which includes 250.000 free Transactions per month. For the Routing API one transaction equals one request with one starting point (no multistop). More information can be found [here](https://developer.here.com/faqs#payment-subscription)
+
+By default HERE will deactivate your account if you exceed the free Transaction limit for the month. You can add payment details to reenable your account as described [here](https://developer.here.com/faqs)
+
+## {% linkable_title Configuration %}
+
+To enable the sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example entry for configuration.yaml
+sensor:
```

Comment:
Only required items in the first example

Reply:
This is the first example....